### DISCUSSION TITLE
SpatieMediaLibraryImageEntry infolist not displaying non-image media uploads

### DISCUSSION THREAD

**Discussion by nea** [filamentphp/filament#12952] [Help]
### Package

Other

### Package Version

v3.2.50

### How can we help you?

Hi there

I am using Spatie Media Library for file uploads of different kinds, images as well as e.g. PDFs.
Uploading both types and displaying on an edit view works fine, can be downloaded and opened.

For a view I created an Infolist with SpatieMediaLibraryImageEntry of the attachments. The image is shown correctly, the PDF just has a "broken" symbol as it is included as <img... tag, which it is not obviously. The url itself is correct behind that.
![image](https://github.com/filamentphp/filament/assets/392035/1b9522e3-1105-4781-b455-79d19348af8e)

Is there a way to also display other than image attachments?
I looked for options on the Entry, but didn't find an obvious one.

Thanks in advance

**Comment by nurmuhammet-ali** [comment]
Would be nice...

**Comment by jeffreyvanhees** [comment]
Maybe you can do something with https://spatie.be/docs/laravel-medialibrary/v11/converting-other-file-types/using-image-generators?
